src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+// eslint-disable-next-line no-unused-vars
+import Calculator from './components/Calculator.js';
```

Comment:
- [ ] Let's try to not disable Linter checks on our codebase as they are meant to help us write a clean and professional codebase. Disabling Linter checks defies its purpose. Kindly fix the errors 👍🏽

Reply:
Done